File: src/server/events/listening.ts
import { ServerConfig, ServerContext } from "../../interfaces";
import { getSchema } from "../utils/schema";
import logger from "../../logger";

export default (context: ServerContext) => {
    return () => {
        logger.info(`Passagaway server listening in: ${getSchema(context.config)}://${context.config.host}:${context.config.port}`)
    }
}

File: src/server/events/listening.py
import logger
from interfaces import ServerConfig, ServerContext
from utils import schema

def start_server(context: ServerContext):
    def listen():
        logger.info(
            f"Passagaway server listening in: {schema.get_schema(context.config)}://"
            f"{context.config.host}:{context.config.port}"
        )

    return listen